src/libraries/System.IO.FileSystem/tests/File/GetSetTimes_String.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class File_GetSetTimes_String : File_GetSetTimes
+    {
+        protected override bool CanBeReadOnly => true;
+        protected override string CreateSymlink(string path, string pathToTarget) => File.CreateSymbolicLink(path, pathToTarget).FullName;
+
+        protected override void SetCreationTime(string path, DateTime creationTime) => File.SetCreationTime(path, creationTime);
+
+        protected override DateTime GetCreationTime(string path) => File.GetCreationTime(path);
+
+        protected override void SetCreationTimeUtc(string path, DateTime creationTime) => File.SetCreationTimeUtc(path, creationTime);
+
+        protected override DateTime GetCreationTimeUtc(string path) => File.GetCreationTimeUtc(path);
+
+        protected override void SetLastAccessTime(string path, DateTime creationTime) => File.SetLastAccessTime(path, creationTime);
+
+        protected override DateTime GetLastAccessTime(string path) => File.GetLastAccessTime(path);
+
+        protected override void SetLastAccessTimeUtc(string path, DateTime creationTime) => File.SetLastAccessTimeUtc(path, creationTime);
+
+        protected override DateTime GetLastAccessTimeUtc(string path) => File.GetLastAccessTimeUtc(path);
+
+        protected override void SetLastWriteTime(string path, DateTime creationTime) => File.SetLastWriteTime(path, creationTime);
+
+        protected override DateTime GetLastWriteTime(string path) => File.GetLastWriteTime(path);
+
+        protected override void SetLastWriteTimeUtc(string path, DateTime creationTime) => File.SetLastWriteTimeUtc(path, creationTime);
+
+        protected override DateTime GetLastWriteTimeUtc(string path) => File.GetLastWriteTimeUtc(path);
+
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotInAppContainer))] // Can't read root in appcontainer
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void PageFileHasTimes()
```

Comment:
Why is this test not in `File_GetSetTimes`? Does it fail somehow when run using the `SafeHandle` APIs?

Reply:
Right. It fails because the system has already opened the `pagefile.sys`.

This can be verified using the [Process Monitor (Sysinternals)](https://docs.microsoft.com/en-us/sysinternals/downloads/procmon).
There a handle for `pagefile.sys` is already opened by the system process (`ntoskernl.exe`).

Reply:
ok, thanks for clarifying.